completion_aggregator/fields.py

```diff
@@ -0,0 +1,67 @@
+"""
+Custom django fields.
+"""
+
+import six
+from opaque_keys.edx.keys import CourseKey, UsageKey
+from opaque_keys import InvalidKeyError
+
+from django.db import models
+
+
+class OpaqueKeyField(models.Field):
```

Comment:
@pomegranited @bradenmacdonald It looks like edx is already working on it in edx/opaque-keys#90, so I would suggest postponing it, rather than blocking on that work.

Reply:
> edx is already working on it in edx/opaque-keys#90

Cool :)